app/src/main/java/com/example/fittracker/ConstantUtils.java

```diff
@@ -19,4 +19,14 @@
     public static final String DATAHANDLER_TYPE = "text/plain";
     public static final Character COMA = ',';
     public static final String MAIL_HOST = "smtp.gmail.com";
+    public static final String WEATHER_BASE_URL = "https://api.openweathermap.org";
```

Comment:
put this in a global variable in gradle pls

Reply:
Done